doc/onboarding.md

```diff
@@ -86,6 +86,10 @@ onboarding session.
       `semver-major` label
     * When adding a `semver-*` label, add a comment explaining why you're adding
       it. Do it right away so you don't forget!
+  * Please add the `ready` label for PRs where the CI was just kicked off and
+    that are good to go (no outstanding review items), pending the CI outcome
```

Comment:
Suggestion: `are good to go (no outstanding review items)` -> `have no outstanding review items`